content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -121,16 +121,23 @@ allows the users to resize the volume by editing the corresponding PVC object.
 The following types of volumes support volume expansion, when the underlying
 Storage Class has the field `allowVolumeExpansion` set to true.
 
-* gcePersistentDisk
-* awsElasticBlockStore
-* Cinder
-* glusterfs
-* rbd
-* Azure File
-* Azure Disk
-* Portworx
-* FlexVolumes
-* CSI  {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< table caption = "Feature state support for various storage types"  >}}
```

Comment:
```suggestion
{{< table caption = "Table of Volume types and the version of Kubernetes they require"  >}}
```

Reply:
@sftim I have made the suggested heading changes. Thank you for the review.